types/ui/contextMenu.d.ts

```diff
@@ -7,72 +7,113 @@
  * @param selector - A CSS selector which activates the context menu.
  * @param menuItems - An Array of entries to display in the menu
  * @param eventName - Optionally override the triggering event which can spawn the menu
- * @param menuItem - Menu items in the array can have the following properties
- * @param name - The displayed item name
- * @param icon - An icon glyph HTML string
- * @param condition - A function which returns a Boolean for whether or not to display the item
- * @param callback - A callback function to trigger when the entry of the menu is clicked
+ *                    (default: `'contextmenu'`)
  */
 declare class ContextMenu {
+  constructor(
+    element: JQuery,
```

Comment:
I just checked `element` definitely *has* to be a `JQery` interface, because `HTMLElement` doesn't have `attr()` or `on()` methods, `JQuery` does though.

Reply:
OK I just checked all variables that are currently typed as JQuery, and on all of them original source calls methods that only exist on JQuery, but not HTMLElement. So this definitely has to be a JQuery.